code/modules/mob/living/silicon/pai/pai_shell.dm

```diff
@@ -103,3 +106,29 @@
 /mob/living/silicon/pai/movement_delay()
 	. = ..()
 	. += 1 //A bit slower than humans, so they're easier to smash
+
+/mob/living/silicon/pai/proc/mob_pickup(mob/living/L)
+	var/obj/item/clothing/head/mob_holder/holder = new(get_turf(src), src, chassis, item_head_icon, item_lh_icon, item_rh_icon)
+	if(!L.put_in_hands(holder))
+		qdel(holder)
+	else
+		L.visible_message("<span class='warning'>[L] scoops up [src]!</span>")
```

Comment:
mob_pickup should be on /mob or /mob/living, and just do nothing or return false for mobs that don't support it.

Reply:
already done